content/en/docs/migration/opentracing.md

```diff
@@ -1,73 +1,47 @@
 ---
 title: Migrating from OpenTracing
-weight: 20
+linkTitle: OpenTracing
 draft: true
 spelling: cSpell:ignore codebases
 ---
 
-The OpenTelemetry project aims to provide backwards compatibility with the
-[OpenTracing][] project, in order to ease migration to
-OpenTelemetry. The intention is to allow OpenTracing instrumentation to be
-recorded using the OpenTelemetry SDK, and to allow both the OpenTelemetry
-and the OpenTracing APIs to be used together in the same codebase.
+Backwards compatibility with [OpenTracing][] has been a priority for the
+OpenTelemetry project from the start. To ease migration, OpenTelemetry supports
+the use of both the OpenTelemetry _and_ OpenTracing APIs in the same codebase.
+This allows OpenTracing instrumentation to be recorded using OpenTelemetry SDKs.
 
-To accomplish this, each OpenTelemetry SDK provides an **OpenTracing Shim**,
-which can be installed in order to connect the SDK to the OpenTracing API.
-Please note that the OpenTracing Shim is not enabled by default.
+To accomplish this, each OpenTelemetry SDK provides an **[OpenTracing
+shim][OT_spec]**, which acts as a bridge between the OpenTracing API and the
```

Comment:
Note the link to the spec's **OT compatibility** page here on the first occurrence of **OpenTracing
shim**.

Reply:
So, the spec doc on the shim is not very useful for end users, it's aim is to be a reference for implementors. I suggest we link to the installation docs for implementation here, as that's what the end users need in order to get started.

Reply:
@tedsuo - that makes sense. Which link would you like for me to use? Or maybe just drop the link?

Reply:
Yeah let's just drop the link here, so that the first links are to the user docs in the next section.

Reply:
Done